src/routing/AppRoutes.tsx

```diff
@@ -100,14 +100,22 @@ const AppRoutes = (props: AppRoutesProps) => {
         </Route>
         <Route element={<RoleBAC rolesAllowed={[USER_ROLES.dataSubmitter, USER_ROLES.chairperson, USER_ROLES.admin]} />}>
           <Route path="/dataset_submissions" element={<DatasetSubmissions />} />
-          <Route path="/data_submission_form" element={<DataSubmissionForm />} />
-          <Route element={<EnvRoute env={envGroups.NON_STAGING} />}>
-            <Route path="/data_submission_form2" element={<DataSubmissionFormV2 />}>
-              <Route path=":studyId" element={<DataSubmissionFormV2 />} />
-            </Route>
-          </Route>
+          {checkEnv(envGroups.NON_STAGING) && (
```

Comment:
Couldn't use the EnvRoute component because we need duplicate paths and ReactRouter had issues when the same paths were possible.

Reply:
I think one way that might get past that is to have the single path, but have a conditional that returns different components inside `element`. I had to do something similar with `SOAcknowledged`.

Reply:
Agreed - I thought about that option but since this is temporary, I didn't think we'd keep them around long and they'd end up being deleted when we're ready to make this "the form."